app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,25 @@
+class LikesController < ApplicationController
+  def new
+    respond_to do |format|
+      format.html { render :new, locals: { like: Like.new } }
+    end
+  end
```

Comment:
- [ ] You do not need this action in your controller because you are to have a button that can enable users like a post whenever the button is clicked. 👍

Reply:
This issue is well resolved! ✔️